tensorflow_addons/seq2seq/attention_wrapper.py

```diff
@@ -1562,7 +1562,31 @@ def _compute_attention(
 
 
 class AttentionWrapper(tf.keras.layers.AbstractRNNCell):
-    """Wraps another RNN cell with attention."""
+    """Wraps another RNN cell with attention.
+
+    Example:
+
+    >>> batch_size = 4
+    >>> max_time = 7
+    >>> hidden_size = 32
+    >>>
+    >>> memory = tf.random.uniform([batch_size, max_time, hidden_size])
+    >>> memory_sequence_length = tf.fill([batch_size], max_time)
+    >>>
+    >>> attention_mechanism = tfa.seq2seq.LuongAttention(hidden_size)
+    >>> attention_mechanism.setup_memory(memory, memory_sequence_length)
```

Comment:
Do we suggest user to call setup_memory() explicitly, or just pass the memory related param in the __init__()?

Reply:
Which one do you think is a better example? I used `setup_memory` because it sets the path on how to build the model first and run it later.

In any case, we add a comment indicating that there is another way to set the memory.